docs/government.md

```diff
@@ -156,6 +156,17 @@ pageClass: routes
 
 <Route author="Fatpandac" example="/gov/huizhou/zwgk/jgdt" path="/gov/huizhou/zwgk/:category?" :paramsDesc="['资讯类别，可以从网址中得到，默认为政务要闻']"/>
 
+## 国家广播电视总局
+
+### 分类
+
+<Route author="yuxinliu-alex" example="/nrta/news/important" path="/nrta/news/:category?" :paramsDesc="['资讯类别， 可从地址中获取']"/>
```

Comment:
This route should be added into `lib/v2/gov` as `.gov`-related websites are added there.